scripts/docker-run.sh

```diff
@@ -29,18 +27,25 @@ fi
 # Variables
 #
 
+DOCKER=${USE_CONTAINER:-docker}
```

Comment:
I think the `REBUILD_CONTAINER_IMAGE` needs to have default initialization here too.

Reply:
So it does, yes, thanks.